packages/fuselage/src/playground/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {
+  LiveProvider,
+  LiveEditor,
+  LiveError,
+  LivePreview,
+} from 'react-live';
+import dracula from 'prism-react-renderer/themes/dracula';
+import duotoneDark from 'prism-react-renderer/themes/duotoneDark';
+import duotoneLight from 'prism-react-renderer/themes/duotoneLight';
+import github from 'prism-react-renderer/themes/github';
+import nightOwl from 'prism-react-renderer/themes/nightOwl';
+import nightOwlLight from 'prism-react-renderer/themes/nightOwlLight';
+import oceanicNext from 'prism-react-renderer/themes/oceanicNext';
+import palenight from 'prism-react-renderer/themes/palenight';
+import shadesOfPurple from 'prism-react-renderer/themes/shadesOfPurple';
+import synthwave84 from 'prism-react-renderer/themes/synthwave84';
+import ultramin from 'prism-react-renderer/themes/ultramin';
+import vsDark from 'prism-react-renderer/themes/vsDark';
```

Comment:
Pick just one.

Reply:
Sorry if I believe in a world where programmers can express themselves using whatever theme they want...

Jk, I guess I just went a little too overboard with it 😝 .

Reply:
To be fair we can rollback when Storybook allow us to switch light and dark mode in the future.